homeassistant/components/alarm_control_panel/services.yaml

```diff
@@ -60,6 +60,17 @@ envisalink_alarm_keypress:
       description: 'String to send to the alarm panel (1-6 characters).'
       example: '*71'
 
+envisalink_alarm_arm_max:
```

Comment:
Move this to a services.yaml file under the envisalink integration component package.